src/Models/review/Transformers/ReviewTransformer.ts

```diff
@@ -0,0 +1,17 @@
+import BaseTransformer from '../../../BaseTransformer';
+import ReviewInterface from '../../../Interfaces/review/Review';
+import Review from '../Review';
+
+export default class ReviewTransformer extends BaseTransformer {
+    mapData(review: ReviewInterface): Review {
+        return new Review({
+            id: review.id,
+            rating: review.rating,
+            createdAt: review.createdAt,
+            updatedAt: review.updatedAt,
+            // has one
+            userId: review.userId,
+            productVariantId: review.productVariantId,
```

Comment:
Sorry, I wrote these a little wrong on the ticket. it should look like:

```suggestion
            // has one
            user: this.includeUser(...),
            productVariant: this.includeProductVariant(...),
```